src/fides/api/graph/traversal.py

```diff
@@ -380,6 +380,18 @@ def edge_ends_with_collection(_edge: Edge) -> bool:
                     [{','.join([str(tn.address) for tn in running_node_queue.data])}]"""
                 )
 
+        # Remove nodes that have custom request fields, since we don't care
+        # if these are reachable or not. They will be used independently
+        # by the Dynamic Email Erasure Connector.
+        # TODO: ideally we'll update the Traversal code to include these
+        # "custom request field datasets" as part of the main graph. This is
+        # a targeted workaround for now.
+        remaining_node_keys = [
+            key
+            for key in remaining_node_keys
+            if not self.traversal_node_dict[key].node.collection.custom_request_fields()
+        ]
```

Comment:
We take the `remaining_node_keys` and filter out the nodes that have custom_request_fields , so we don't error because they're unreachable. This was a workaround proposed by @galvana so that we could deliver this feature in time, but in the future we'd want to incorporate custom request fields into the graph building logic